analyzer/codechecker_analyzer/cmd/analyzer_version.py

```diff
@@ -32,6 +32,9 @@ def __init__(self):
         self.git_hash = context.package_git_hash
         self.git_tag = context.package_git_tag
 
+    def is_release_candidate(self):
+        return 'rc' in self.version
```

Comment:
Shouldn't it be
```suggestion
        return 'rc' in self.version and self.version['rc']
```
?

And this function could go under some codechecker_common/util module, I think.

Reply:
Oh, this is not a dict, but a string. Sorry, then this comment is irrelevant.

Reply:
self.version is a string here, I believe

Reply:
This would indeed be better in codechecker_common, in fact, this entire version chicanery could be, but that's outside the scope of this patch :)